health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/accessbilityPopUP.js

```
@@ -62,76 +70,90 @@ const AccessibilityPopUp = ({ onClose, census }) => {
-              option={state.villageRoadCondition}
-              optionKey="name"
-              selected={dropdown1Value}
-              select={(value) => handleDropdownChange(value, "dropdown1")}
-              t={(label) => label} // Replace with translation function
-            />
-          </LabelFieldPair>
+    <>
+      <PopUp
+        onClose={onClose}
+        heading={t(`HCM_MICROPLAN_VILLAGE_ACCESSIBILITY_LABEL`)}
+        children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)